gallery/src/demos/demo-hui-alarm-panel-card.ts

```diff
@@ -69,9 +81,10 @@ class DemoAlarmPanelEntity extends PolymerElement {
     };
   }
 
-  public ready() {
+  public async ready() {
```

Comment:
Don't make functions that we override async if the original one is not.

Reply:
Oops, that's a left over